### ISSUE TITLE
Problem with automatically recognized meeting room name

### ISSUE THREAD

**Issue by jfan-dolby** [voxeet/voxeet-io-web#2]
We have requested several environments of the app and some of them require a trailing environment such as https://dev.voxeet.trydcc.com/staging/. It seems that when this link is used, the join form mistakenly takes /staging/ as the meeting room name, this is unexpected and should be fixed.

The join meeting form should take the last part of the URL without trailing slash as the intended meeting room name, if it didn't detect that name, it should present the user to manually enter the meeting room name.

Basically this problem prevents us from testing against the non-production Voxeet environment.
